### ISSUE TITLE
Can't type in fields & Dialog failed

### ISSUE THREAD

**Issue by eltoredo** [ckeditor/ckeditor4#3555]
I CAN'T CHANGE THE TITLE AS I SOLVED ONE ISSUE

## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Initialize any CKEditor and try to fill a field in any of the basic plugins

### Expected result

I would like to be able to fill the fields when I want to load an image for example.

### Actual result

When I click on Font Awesome, for example, the new plugin button that I installed, I can't fill the fields.

## Other details

* Browser: Firefox last version
* OS: Windows 10
* CKEditor version: CKEditor 4
* Installed CKEditor plugins: Full + Font Awesome one

I wonder if I'm not able to search for a result efficently as I think that this common issue should have lots of duplicates on internet. I've seen one or two posts about it but nothing related to me or that fixed the problem.

Thanks for reading!

**Comment by jacekbogdanski** [comment]
I'm sorry but honestly, I don't understand the issue you are struggling with. Could you provide reproduction steps with a demo application showing up what's going wrong? Some gif image also would work.

Note that we are not able to fix 3rd party plugins as we simply don't own code, so if the issue is caused by Font Awesome plugin, please contact plugin owner for help.

**Comment by eltoredo** [comment]
Indeed, it wasn't very clear, sorry.

What I mean is that whenever I click on a button, wether it's a basic plugin or an added one, I can't fill any of the given fields, I can't edit anything. It's like there is no keyboard focus when I want to fill an input.

Here are two screenshots: 
https://imgur.com/BB8RDfB - When I can't write anything in the inputs, no focus
https://imgur.com/BufQ2y2 - When I try to validate, because nothing has been written

It's as simple as that, if you really need a gif of me trying to type in the fields, I'll do it.
I hope it's more clear, thanks for your quick answer and have a good day!

Tanguy.

**Comment by jacekbogdanski** [comment]
I tried to reproduce the issue with the same dialog (image) using the latest Firefox on Windows with default CKEditor4 config, and as you can see on gif image, everything works smoothly:

![dialog](https://user-images.githubusercontent.com/10279586/66634299-0be81580-ec0d-11e9-8d4e-b9dd3d8720d7.gif)

So your issue is probably caused by 3rd party plugins or different editor configuration.

Could you verify if you are able to reproduce the issue with one of our builds (standard or full) without 3rd party plugins? Also, it would help if you provide your CKEditor4 configuration.

**Comment by eltoredo** [comment]
Indeed, it works perfectly fine in a new and purified page.

This is my CKEditor config: https://hastebin.com/ukomopizay.js (the full one, + Font Awesome one for the moment)

And this is the parts of my code that makes it work (it's in a modal so it may be the problem, using Bootstrap): 

HTML: https://hastebin.com/etodalahor.xml
JS: https://hastebin.com/anoworeser.js

It should be it, if I don't miss any code.

Thanks again for your quick answers!

**Comment by eltoredo** [comment]
Little up!

**Comment by jacekbogdanski** [comment]
I still can't reproduce the issue using your config. We are not able to help here with custom application integrations, and it seems like your issue is caused by putting editor into a modal dialog. I can recommend leaving your question on [Stack Overflow](https://stackoverflow.com/questions/tagged/ckeditor) where our team and the community assist other users in solving their issues.

**Comment by eltoredo** [comment]
I opened a question on Stack Overflow as you advised me to do: https://stackoverflow.com/questions/58482267/ckeditor-i-cant-fill-any-fields-no-focus-on-inputs

At the end, I wrote something that might help to find a solution if never you're still up to help me.

This is what I'm talking about: I hit the Escape key just when I clicked on one of the plugins that require to enter text, like the image one. It made the modal close but the plugin form still opened itself and I was able to type in the fields. If I fill in the fields and try to press 'OK', I get an error 'b is undefined'.

Screenshot: https://imgur.com/lhgSrTs
